.circleci/config.yml

```
@@ -51,7 +51,10 @@ jobs:
 workflows:
   main:
     jobs:
-      - validate
+      - validate:
+          filters:
+            tags:
+              only: "/^v.*/"
```

Don't we want `validate` job to be run for any tags.
```suggestion
      - validate:
          filters:
            tags:
              only: "/.*/"
```

---

We could, but since `publish_validate` will ignore tags that are not starting with `v` it is worth it to have `validate` handle all type of tags?